### PR TITLE
Positronics no longer sit inside MMIs

### PR DESCRIPTION
**Pull request by zxaber**
## About The Pull Request
Fixes an overlay bug where Posis were placing the MMI overlay on themselves when gaining a ghost.

## Why It's Good For The Game
image

## Changelog
:cl:
fix: Positronics no longer have MMIs visually spawn on them.
/:cl:

### CODE REVIEW

**Comment by PKPenguin321** [comment]
Aww you're taking away their smile

File: code/modules/mob/living/brain/MMI.dm [hunk 1]
```diff
@@ -24,6 +24,9 @@
 
 /obj/item/mmi/update_overlays()
 	. = ..()
+	. += add_mmi_overlay()
+
+/obj/item/mmi/proc/add_mmi_overlay()
 	if(brainmob && brainmob.stat != DEAD)
 		. += "mmi_alive"
```
Comment (MrPerson) [inline code/modules/mob/living/brain/MMI.dm]: This proc is doing . += "string" but . is still null. This works but it's misleading.
```suggestion
		return "mmi_alive"
```

**Comment by Arathianen** [comment]
This is a bug? I actually really liked it... it visually showed me very clearly when a positronic brain became active.

Honestly, I would prefer keeping this "bug".

**Comment by zxaber** [comment]
Positronics are already visually active when the blue lights start tracing along the faces. The MMI being overlaid was certainly not intentional.

It doesn't even make any sense. Why does a cube need to connect to an interface normally meant for human brains? Why isn't the interface built in anymore? Where does the MMI even come from? These are all serious questions that need to be answered if the overlay was to be kept.

**Comment by Arathianen** [comment]
I know it visually showed before, but it was easy to miss. Even for experienced robos.

I don't really know how it makes sense, but perhaps this is a good opportunity to resprite the active bit instead of putting it back to where it was?

**Comment by Fikou** [comment]
> I know it visually showed before, but it was easy to miss. Even for experienced robos.

you being blind doesnt make it easy to miss

**Comment by TheMidnightRose** [comment]
> I know it visually showed before, but it was easy to miss. Even for experienced robos.
> 
Awake Posis generally start yelling for attention after being ignored for a couple of minutes, they even get the Common radio to do so.

**Pull request merged**